README.md

```
@@ -74,14 +71,48 @@ uv run python -m backend.main
 Terminal 2 (Frontend):
 ```bash
 cd frontend
-npm run dev
+pnpm run dev
 ```
 
 Then open http://localhost:5173 in your browser.
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)